editor/doc_tools.cpp

```diff
@@ -394,13 +394,22 @@ void DocTools::generate(bool p_basic_types) {
 					method.qualifiers += " ";
 				}
 				method.qualifiers += "const";
-			} else if (E->get().flags & METHOD_FLAG_VARARG) {
+			}
```

Comment:
This seemed wrong somehow, not quite sure why it was like this, so I changed it.